api/v1beta1/install/install.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+// Package install installs the experimental API group, making it available as
+// an option to all of the API encoding/decoding machinery.
+package v1beta1
```

Comment:
```suggestion
package install
```

Maybe, I'm wrong, but usually the name of the package is the same as the folder.